packages/core/src/App/Containers/Modals/ready-to-deposit-modal/ready-to-deposit-modal.jsx

```diff
@@ -0,0 +1,56 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import { Dialog } from '@deriv/components';
+import { localize } from '@deriv/translations';
+import { connect } from 'Stores/connect';
+import './ready-to-deposit-modal.scss';
+
+const ReadyToDepositModal = ({ is_open, onClose, disableApp, enableApp, openRealAccountSignup, is_eu_user }) => {
```

Comment:
@thisyahlen-deriv It's a newly added component, Why `jsx` and not using the `useStore` hook to have type-safety? 🤔

Reply:
Can do